squidpy/pl/_graph.py

```diff
@@ -29,10 +29,16 @@
 __all__ = ["centrality_scores", "interaction_matrix", "nhood_enrichment", "ripley", "co_occurrence"]
 
 
-def _get_data(adata: AnnData, cluster_key: str, func_name: str, **kwargs: Any) -> Any:
+def _get_data(adata: AnnData, cluster_key: str, func_name: str, attr: str = "uns", **kwargs: Any) -> Any:
```

Comment:
I think we don't need can be remove?

Reply:
Here i'm not quite sure how it's related to the plotting function. But yes, it's not needed for the plotting function.